pkg/csi_driver/node.go

```diff
@@ -266,6 +266,9 @@ func (s *nodeServer) NodePublishVolume(ctx context.Context, req *csi.NodePublish
 		return nil, status.Errorf(codes.Internal, "mkdir failed for path %q: %v", targetPath, err)
 	}
 
+	disallowedFlags, _ := s.driver.generateDisallowedFlagsMap(gcsFuseSidecarImage)
```

Comment:
Here, return the error as internal error (this should never happen, but if it does, its internal error). We shouldn't ignore errors returned from functions